packages/sign-client/test/lifecycle/lifecycle.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { getSdkError } from "@walletconnect/utils";
+import {
+  initTwoClients,
+  testConnectMethod,
+  deleteClients,
+  uploadCanaryResultsToCloudWatch,
+  throttle,
+  publishToStatusPage,
+} from "../shared";
+import { TEST_RELAY_URL } from "../shared/values";
+import { describe, it, expect, afterEach } from "vitest";
+
+const environment = process.env.ENVIRONMENT || "dev";
+
+const timeout = environment === "prod" ? 610_000 : 70_000;
```

Comment:
Think it makes sense to break these magic numbers out into their own labelled `const`, also since there's hardcoded references below that should vary by env, i.e.:

```ts
const PROD_TIMEOUT_IN_MS = 610_000
const PROD_TIMEOUT_IN_MS = 70_000
```

Reply:
Actually disregard, not that necessary anymore, found a simpler way to deal with the timeout reference further down 👍